glib-macros/src/lib.rs

```diff
@@ -249,6 +250,31 @@ pub fn genum_derive(input: TokenStream) -> TokenStream {
     gen.into()
 }
 
+/// Derive macro for defining a GLib error and its [`ErrorDomain`] trait.
+///
+/// # Example
+///
+/// ```
+/// use glib::prelude::*;
+/// use glib::subclass::prelude::*;
+///
+/// #[derive(Debug, Copy, Clone, glib::GError)]
+/// #[gerror(domain_name = "ExFoo")]
+/// enum Foo {
```

Comment:
why do you think it's necessary? It's the rust implementation after, doesn't have an influence on the ffi, right? Afaict, "self as i32" in code() is the only case where it would matter, but it should be fine whatever representation, isn't it?

Reply:
Right, if you always go via `FromGlib` / `ToGlib` it's irrelevant indeed